src/attestation/azext_attestation/manual/custom.py

```diff
@@ -188,7 +188,7 @@ def get_policy(cmd, client, attestation_type, resource_group_name=None, provider
 
     if token:
         import jwt
-        policy = jwt.decode(token, verify=False).get('x-ms-policy', '')
+        policy = jwt.decode(token, algorithms=['RS256'], options={"verify_signature": False}).get('x-ms-policy', '')
```

Comment:
There should be some other places using "jwt.decode", please update them as well.